pages/news/2020-05-15-you-dont-need-to-pay-for-mixxx.html

```diff
@@ -0,0 +1,77 @@
+title: You don't need to pay for Mixxx!
+author: Holzhaus
+date: 2020-05-15 13:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+It recently came to our attention that some sellers distribute boxed versions of Mixxx on Ebay and Amazon.
+While it's not illegal to do this per se, we are not affiliated with these sellers and neither earn money from this nor endorse this in any way.
+Instead, you can download the latest version of Mixxx from the [download page on our website]({% url "/download.html" %}) for free.
+
+If you want to support Mixxx financially, consider donating - we'll appreciate it!
+You can use the *Donate* button on the download page.
+
+
+### Mixxx was, is and always will be free
+
+In contrast our commercial competitors like Serato or VirtualDJ that may offer "free" featured-reduced versions of their Software, the Mixxx developers do not impose any artificial restrictions on their software.
```

Comment:
I prefer not to call out competitors by name, esp because we are writing code that interoperates with their software.  Best not to make enemies unnecessarily.

Reply:
Agreed.